app/javascript/components/subject/PauseNotifications.js

```diff
@@ -32,52 +32,79 @@ class PauseNotifications extends React.Component {
+        <span>{text}</span>
+      </ReactTooltip>
+    );
+  }
+
   render() {
     return (
       <React.Fragment>
         {!this.props.patient.pause_notifications && (
-          <Button
-            id="pause_notifications"
-            className="mr-2"
-            disabled={this.state.loading}
-            onClick={() =>
-              this.handleSubmit(
-                "You are about to change this monitoree's notification status to paused. This means that the system will stop sending the monitoree symptom report requests until notifications are resumed by a user."
-              )
-            }>
-            <i className="fas fa-pause"></i> Pause Notifications
-            {this.state.loading && (
-              <React.Fragment>
-                &nbsp;<span className="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-              </React.Fragment>
-            )}
-          </Button>
+          <React.Fragment>
+            <span data-for="notifications-tooltip" data-tip="">
+              <Button
+                id="pause_notifications"
+                className="mr-2"
+                disabled={this.state.disableAndDisplayTooltip || this.state.loading}
+                onClick={() =>
+                  this.handleSubmit(
+                    "You are about to change this monitoree's notification status to paused. This means that the system will stop sending the monitoree symptom report requests until notifications are resumed by a user."
+                  )
+                }>
+                <i className="fas fa-pause"></i> Pause Notifications
+                {this.state.loading && (
+                  <React.Fragment>
+                    &nbsp;<span className="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
+                  </React.Fragment>
+                )}
+              </Button>
+            </span>
+            {this.state.disableAndDisplayTooltip && this.renderTooltip()}
+          </React.Fragment>
         )}
         {this.props.patient.pause_notifications && (
-          <Button
-            id="pause_notifications"
-            className="mr-2"
-            disabled={this.state.loading}
-            onClick={() =>
-              this.handleSubmit(
-                "You are about to change this monitoree's notification status to resumed. This means that the system will start sending the monitoree symptom report requests unless notifications are paused by a user or the record is closed."
-              )
-            }>
-            <i className="fas fa-play"></i> Resume Notifications
-            {this.state.loading && (
-              <React.Fragment>
-                &nbsp;<span className="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
-              </React.Fragment>
-            )}
-          </Button>
+          <React.Fragment>
```

Comment:
This is duplicated code. Only difference is "resumed" vs "paused" and "Resume" vs "Pause' which we can easily handle. I'd ask you to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) this up

Reply:
done